dialogue-annotations/src/main/java/com/palantir/dialogue/annotations/Request.java

```diff
@@ -86,6 +86,8 @@
     @Target(ElementType.PARAMETER)
     @interface PathParam {
         Class<? extends ParamEncoder<?>> encoder() default DefaultParamEncoder.class;
+
+        Class<? extends ListParamEncoder<?>> listEncoder() default UnsetListParamEncoder.class;
```

Comment:
nit: no reason to create a new type, we could use the `DefaultListParamEncoder` marker interrface -- annotation processors can differentiate between default values and values that are explicitly set.